examples/amp-story/amp-story-shopping.html

```diff
@@ -17,21 +17,65 @@
         publisher="AMP Story Shopping"
         publisher-logo-src="example.com/logo.png"
         poster-portrait-src="example.com/poster.jpg">
-      <amp-story-page id="default">
-        <amp-story-shopping-config layout='nodisplay'></amp-story-shopping-config>
+      <amp-story-page id="example">
+        <!--
+          This is an example of a JSON config retrieved from a source file.
+          To use the inline JSON config, remove the src attribute.
+        -->
+        <amp-story-shopping-config layout='nodisplay' src="/examples/amp-story/shopping/remote.json" >
+          <script type="application/json">
```

Comment:
Please add another page to this story that uses inline JSON with no src attribute.
It can have a config with different product data.